docs/user/dashboard/explore-dashboard-data.asciidoc

```diff
@@ -0,0 +1,20 @@
+[[explore-dashboard-data]]
+== Explore dashboard data
+
+Get a closer look at your data by inspecting elements and using drilldown actions.
```

Comment:
There's more information coming for drilldown actions for 7.10.  i think they should have their own section.